database/schema.sql

```diff
@@ -52,36 +62,8 @@ CREATE TABLE earthquakes (
     FOREIGN KEY (magtype_id) REFERENCES magtypes(magtype_id)
 );
 
-
-INSERT INTO alerts
-    (alert_value)
-VALUES 
-     ('green'), ('yellow'), ('orange'), ('red')
-;
-
-INSERT INTO networks
-    (network_name)
-VALUES 
-     ('ak'), ('at'), ('ci'), ('hv'), ('ld'), 
-     ('mb'), ('nc'), ('nm'), ('nn'), ('pr'), 
-     ('pt'), ('se'), ('us'), ('uu'), ('uw')
-;
-
-INSERT INTO types
-    (type_value)
-VALUES 
-     ('earthquake'), ('quarry')
-;
-
-INSERT INTO statuses
-    (status_value)
-VALUES 
-     ('automatic'), ('reviewed'), ('deleted')
-;
-
-INSERT INTO magtypes
-    (magtype_value)
-VALUES 
-     ('md'), ('ml'), ('ms'), ('mw'), ('me'), 
-     ('mi'), ('mb'), ('mlg')
-;
+INSERT INTO alerts (alert_value) VALUES ('green'), ('yellow'), ('orange'), ('red');
+INSERT INTO networks (network_name) VALUES ('ak'), ('at'), ('ci'), ('hv'), ('ld'), ('mb'), ('nc'), ('nm'), ('nn'), ('pr'), ('pt'), ('se'), ('us'), ('uu'), ('uw');
+INSERT INTO types (type_value) VALUES ('earthquake'), ('quarry');
+INSERT INTO statuses (status) VALUES ('automatic'), ('reviewed'), ('deleted');
+INSERT INTO magtypes (magtype_value) VALUES ('md'), ('ml'), ('ms'), ('mw'), ('me'), ('mi'), ('mb'), ('mlg');
```

Comment:
```suggestion
-- Insert values into the alerts table
INSERT INTO alerts (alert_value) VALUES 
    ('green'), 
    ('yellow'), 
    ('orange'), 
    ('red');

-- Insert values into the networks table
INSERT INTO networks (network_name) VALUES 
    ('ak'), 
    ('at'), 
    ('ci'), 
    ('hv'), 
    ('ld'), 
    ('mb'), 
    ('nc'), 
    ('nm'), 
    ('nn'), 
    ('pr'), 
    ('pt'), 
    ('se'), 
    ('us'), 
    ('uu'), 
    ('uw');

-- Insert values into the types table
INSERT INTO types (type_value) VALUES 
    ('earthquake'), 
    ('quarry');

-- Insert values into the statuses table
INSERT INTO statuses (status) VALUES 
    ('automatic'), 
    ('reviewed'), 
    ('deleted');

-- Insert values into the magtypes table
INSERT INTO magtypes (magtype_value) VALUES 
    ('md'), 
    ('ml'), 
    ('ms'), 
    ('mw'), 
    ('me'), 
    ('mi'), 
    ('mb'), 
    ('mlg');

```

Reply:
may be clearer but up to you if you want to add these changes

Reply:
Hmm, I think it's quite readable the current way, I'll ask what everyone else thinks too